### PR TITLE
Password modal badge

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
###  **Implemented the Status Badge on the Password modal When verifying new MT5 Account**

Screenshot 2024-09-11 at 2 27 25 PM

Screenshot 2024-09-11 at 2 27 47 PM

### CODE REVIEW

File: packages/components/src/components/status-badge/status-badge.scss [hunk 1]
```diff
@@ -29,7 +29,7 @@
         &--migrated_without_position,
         &--under_maintenance,
         &--unavailable {
-            border: 1.5px solid var(--status-warning);
+            border: 1.5px solid var(--c);
```
Comment (amina-deriv) [inline packages/components/src/components/status-badge/status-badge.scss]: what is c here?
also this effects all badges @Dhruv-deriv

Comment (Dhruv-deriv) [reply packages/components/src/components/status-badge/status-badge.scss]: typo error

File: packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx [hunk 2]
```diff
@@ -29,15 +59,18 @@ const ListItem = observer(({ id, text, status, route }: TListItemProps) => {
...
+            </div>
             {status === AUTH_STATUS_CODES.NONE || (id === 'tax' && status === 0) ? (
                 <LabelPairedChevronRightCaptionBoldIcon />
             ) : (
-                <Text size={isMobile ? 'xxs' : 'xs'} line_height='xl'>
-                    <Localize i18n_default_text={status} />
-                </Text>
+                <div className='verification-docs-list-modal__card'>
+                    <StatusBadge account_status={status} icon={badge_icon} text={badge_text} icon_size={badge_size} />
+                    <LabelPairedChevronRightCaptionBoldIcon fill='#c7c7c7' />
```
Comment (yauheni-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx]: just to clarify, is this color on design? why can't we use some of existing grey colors and pass here it as a variable?

**Review by yauheni-deriv — APPROVED** [review]

**Review by suisin-deriv — APPROVED** [review]

File: packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx (hunk 2 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx]: resolved 👍

File: packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx [hunk 3]
```diff
@@ -16,7 +16,37 @@ type TListItemProps = {
...
+                text: <Localize i18n_default_text='Verified' />,
+                icon: 'IcMt5Success',
+                icon_size: '18',
+            };
+        case AUTH_STATUS_CODES.PENDING:
+            return {
+                text: <Localize i18n_default_text='In review' />,
+                icon: 'IcMt5Pending',
+            };
+        case AUTH_STATUS_CODES.REJECTED || AUTH_STATUS_CODES.SUSPENDED:
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx]:
```
case AUTH_STATUS_CODES.REJECTED :
case AUTH_STATUS_CODES.SUSPENDED:
 return {
                text: <Localize i18n_default_text='Failed' />,
                icon: 'IcMt5Failed',
                icon_size: '18',
            };
```

Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/verification-docs-list-modal.tsx]: DONE 👍

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**